tests/unit/user/route-test.js

```diff
@@ -0,0 +1,18 @@
+import {module, test} from 'qunit';
+import {setupTest} from 'ember-qunit';
+
+module('Unit | Route | user', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    const route = this.owner.lookup('route:user');
+    assert.ok(route);
+  });
+
+  test('navigate function', function (assert) {
+    const route = this.owner.lookup('route:user');
+    const stub = sinon.stub(route, 'transitionTo');
```

Comment:
use `sinon.spy` instead I think: `sinon.stub` will totally replace the `transitionTo` method with a sinon Object, I think - hence the error

Reply:
examples in https://github.com/ConnectedHomes/ember-commons/search?q=sinon.spy&unscoped_q=sinon.spy

Reply:
(or you have `sinon.stub().returns(aMethod)` otherwise but probably overkill here)

Reply:
Even changing it to sinon.spy I get the same issue.

Reply:
ok my bad